src/SFA.DAS.Tools.Servicebus.Support.Web/Views/MessageList/Index.cshtml

```diff
@@ -4,21 +4,105 @@
 @section Scripts
 {
     <script src="~/lib/bootstrap-table/bootstrap-table.min.js"></script>
-
-   <script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.11/lodash.min.js"></script>
+    <script>
 
         let baseUrl = "@(Url.Action("Index", "MessageDetails"))";
 
         function operateFormatter(value, row, index) {
             return '<a class="detail" href=' + (baseUrl + "/?id=" + row.id) +' title ="Detail">Detail</a>';
         };
 
+
+        //maintain selected items
```

Comment:
Now this script has grown we should look to extract it out into a .js file if possible